README.md

```diff
@@ -1,5 +1,7 @@
 ![idyll-text](https://cloud.githubusercontent.com/assets/1074773/24593896/95730fba-17dc-11e7-82dd-ae7335f205b6.png)
 
+[![Backers on Open Collective](https://opencollective.com/idyll/backers/badge.svg)](#backers) [![Sponsors on Open Collective](https://opencollective.com/idyll/sponsors/badge.svg)](#sponsors)
+
 *[Online Editor](https://idyll-lang.github.io/editor)* 
 
 *See https://idyll-lang.github.io/ for full documentation*.
```

Comment:
Can we move the badges to come right directly after this "... for full documentation" line?